Havit.Blazor.Components.Web.Bootstrap/Forms/Autosuggests/Internal/HxAutosuggestItems.razor.cs

```diff
@@ -8,9 +8,34 @@ public partial class HxAutosuggestItems<TItem>
 
 		[Parameter] public RenderFragment<TItem> ItemTemplate { get; set; }
 
+		/// <summary>
+		/// Visually highlights the first suggestion.
+		/// </summary>
+		[Parameter] public bool HighlightFirstSuggestion { get; set; }
+
 		[Parameter] public RenderFragment EmptyTemplate { get; set; }
 		[Parameter] public string CssClass { get; set; }
 
+		private int hasFocusCount = 0;
```

Comment:
Why do we count the focuses when we are interested only in true/false? Is there any glitch behind that?